pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
 # Minimum requirements for the build system to execute.
-requires = ["setuptools", "wheel", "jupyter-packaging", "jupyterlab"]
```

Comment:
condition on python2 only

Reply:
looks like this:
https://github.com/jpmorganchase/nbcelltests/blob/master/pyproject.toml#L6

` ; python_version < '3'`